VTEX - Payment Provider Protocol.json

```diff
@@ -2104,169 +2330,45 @@
                     "Configuration Flow"
                 ],
                 "summary": "Get Credentials",
-                "description": "Get the credentials of merchant.",
+                "description": "Retrieves the credentials of merchant. \r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
What credentials? Why would this credentials need to be retrieved?

Pelase be more specific about the context and usage.